docs/docs/rest-api/public/api/v2/apps.raml

```diff
@@ -90,6 +90,16 @@
       Note&#58;  This operation will create a deployment. The operation finishes, if the deployment succeeds.
        You can query the deployments endoint to see the status of the deployment.
     is: [ secured, deployable ]
+    queryParameters:
+      updateExisting:
```

Comment:
i'm confused by this parameter name. aren't we updating via PUT or PATCH? how does `updateExisting` imply "I'd like to partially update"?

suggestions: `partialUpdate`, `patchUpdate`, `applyAsPatch`